lib/global-styles.php

```diff
@@ -135,17 +182,6 @@ function_exists( 'gutenberg_is_edit_site_page' ) &&
 		$settings['styles'][] = $block_styles;
 	}
 
-	$settings['__experimentalFeatures'] = $consolidated->get_settings();
```

Comment:
This is now the `gutenberg_global_styles_experimental_settings_5_8` responsibility, which follows suit on the core PR does. See https://github.com/WordPress/wordpress-develop/pull/1262 for whether we should remove these or retrofit the data for backward-compatibility.

Reply:
We landed on always re-creating the settings in the plugin.